daemon/mgr/container.go

```diff
@@ -955,10 +955,6 @@ func (mgr *ContainerManager) stop(ctx context.Context, c *Container, timeout int
 		return nil
 	}
 
-	if timeout == 0 {
```

Comment:
You can have a look at `ProbeContainer` function, if timeout is 0, it will never return `errtypes.ErrTimeout` message, but you can see ctrd/containerd.go L455, which only call SIGKILL when get `errtypes.ErrTimeout`. 
In other words, if just delete this code directly, it will never delete those containers which ignore `SIGTERM singal`!